src/pool/srv_layout.c

```diff
@@ -49,6 +46,9 @@ RDB_STRING_KEY(ds_pool_prop_, nhandles);
 /** pool handle KVS */
 RDB_STRING_KEY(ds_pool_prop_, handles);
 
+/** Security credential handle KVS */
+RDB_STRING_KEY(ds_pool_sec_, handles);
+
```

Comment:
Is this being used somewhere? Or just an artifact of an earlier implementation?

Reply:
Good catch. Its an artifact of an earlier implementation.